doc/manual/src/release-notes/rl-next.md

```diff
@@ -1,4 +1,16 @@
 # Release X.Y (202?-??-??)
 
+
+* Binary cache stores now have a setting `compression-level`.
+
+* `nix develop` now has a flag `--unpack` to run `unpackPhase`.
+
+* Lists can now be compared lexicographically using the `<` operator.
+
+* New built-in function: `builtins.groupBy`, with the same functionality as
+  Nixpkgs' `lib.groupBy`, but faster.
+
+* Nix now searches for a flake.nix up until git or filesystem boundary.
+
```

Comment:
```suggestion
* The Nix cli now searches for a flake.nix up until the root of the current git repository or a filesystem boundary rather than just in the current directory
```